config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1161,7 +1162,6 @@ presubmits:
         args:
         - --deployment=node
         - --env=KUBE_SSH_USER=core
-        - --gcp-project-type=node-e2e-project
```

Comment:
Not entirely sure why but it seems that presubmits don't require this project-type.